docs/user_guide/models/totalvi.rst

```diff
@@ -0,0 +1,222 @@
+===========
+totalVI
+===========
+
+**totalVI** [#ref1]_ posits a flexible generative model of CITE-seq RNA and protein data that can subsequently
+be used for many common downstream tasks.
+
+The advantages of totalVI are:
+
+    + Comprehensive in capabilities.
+
+    + Scalable to very large datasets (>1 million cells).
+
+The disadvantages of totalVI include:
+
+    + Effectively requires a GPU for fast inference.
+
+    + Difficult to understand the balance between RNA and protein data in the low-dimensional representation of cells.
+
+.. topic:: Tutorials:
+
+ - :doc:`/tutorials/notebooks/totalVI`
+ - :doc:`/tutorials/notebooks/cite_scrna_integration_w_totalVI`
+ - :doc:`/tutorials/notebooks/scarches_scvi_tools`
+
+
+Preliminaries
+==============
+totalVI takes as input a scRNA-seq gene expression matrix of UMI counts :math:`X` with :math:`N` cells and :math:`G` genes
+along with a paired matrix of protein abundance (UMI counts) :math:`Y`, also of :math:`N` cells, but with :math:`T` proteins.
+Thus, for each cell, we observe both RNA and protein information.
+Additionally, a design matrix :math:`S` containing :math:`p` observed covariates, such as day, donor, etc, is an optional input.
+While :math:`S` can include both categorical covariates and continuous covariates, in the following, we assume it contains only one
+categorical covariate with :math:`B` categories, which represents the common case of having multiple batches of data.
+
+
+Generative process
+========================
+
+.. figure:: figures/totalvi_graphical_model.svg
```

Comment:
would suggest moving this further below once you have defined the variables

Reply:
also noticed that this image doesnt include the global parameters

Reply:
going to leave this as is, we typically haven't included global params in these graphical models